Comments Module 1.txt

```diff
@@ -1,3 +1,9 @@
 https://github.com/simonbaily/DevOps-Course-Starter/pull/1
 
 Dont understand comment #4 about IDs
+
+Swagger. Is there one for Trello? How would I find out?
```

Comment:
If an API provides a web interface like Swagger, you'd usually have to search their docs - sadly I'm not aware of one for Trello! They try to provide code snippets to try out individual calls, but you'll have to use e.g. curl/python/postman or similar tools